CONTRIBUTING.md

```diff
@@ -0,0 +1,231 @@
+# Contributing to Prism Themes
+
+Thank you for coming on board!
+
+There are a few ways you can contribute to Prism Themes, such as:
+
+-   [Filing bug reports](#filing-bug-reports)
+-   [Discussing the project](#discussing-the-project)
+-   [Submitting Pull Requests](#submitting-pull-requests), whether it's for documentation or code
+-   [Creating new themes](#creating-new-themes)
+
+## Filing bug reports
+
+If you run into an error or a bug while using a theme from this repository:
+
+1.  Confirm that the bug lies with the theme and not Prism or one of its plugins. You can check by swapping the theme out for another theme. If the issue lies with Prism or one of its plugins, please head over to the [main repository](https://github.com/PrismJS/prism) instead.
+    -   In addition, please be sure that you only have one theme loaded! Having multiple themes loaded may result in conflicting or unexpected styles.
+
+2.  Check [existing issues (both open and closed)](https://github.com/PrismJS/prism-themes/issues?q=is%3Aissue) to see if it had previously been reported and/or resolved. If an existing issue exists, please leave a comment on it instead of creating a new issue.
+
+3.  [Open a new issue](https://github.com/PrismJS/prism-themes/issues/new/choose) with a clear and concise title, and include any relevant information such as:
+    -   The name of the theme you're having an issue with
+    -   Any plugins that you're using, if applicable
+    -   Screenshots and/or a reproducible sample
+    -   Browser information
+
+### Security issues
+
+In the case of security issues, please check our [Security Policy](SECURITY.md) for details instead.
+
+## Discussing the project
+
+Perhaps you have some thoughts on Prism Themes, whether it's the state of the code, documentation, features you think might be useful, or simply want to know when the next release will be because you really like the latest theme and want to use it! Please [open a new issue](https://github.com/PrismJS/prism-themes/issues/new/choose) and let us know!
+
+## Submitting Pull Requests
+
+Whether it's documentation or code, Prism Themes welcomes Pull Requests! (If you're specifically looking to contribute a brand new theme, the next section on [Creating new themes](#creating-new-themes) would be more relevant.)
+
+1.  Fork this repository and create a new branch. It is possible that you do not have to clone it to your machine if you do not have to regenerate theme screenshots. If you need help at any point, please reach out by [opening a new issue](https://github.com/PrismJS/prism-themes/issues/new/choose)!
+
+2.  Make the changes.
+
+3.  Commit the changes, and include clear and concise commit messages when doing so.
+
+4.  Once you're done making the changes, [submit a Pull Request](https://github.com/PrismJS/prism-themes/compare)!
+    -   If your Pull Request resolves an open issue (or more), please include `closes #issuenumber` (for each issue) in the description.
```

Comment:
Might as well link the relevant documentation. I.e. I used to make quite a few PRs before I knew closing keywords were a thing.

```suggestion
    -   If your Pull Request resolves an open issue (or more), please include [`closes #issuenumber`](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) (for each issue) in the description.
```